packages/addons-website/scss/components/_website-switcher.scss

```diff
@@ -0,0 +1,61 @@
+@import '../prefix';
+
+.#{$prefix}--website-switcher {
+  position: fixed;
+  right: 0;
+  width: 0;
+  background-color: $ibm-colors__gray--100;
+  height: 100%;
+  z-index: 1000;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2);
+  will-change: width;
+  transition: width 0.11s cubic-bezier(0.2, 0, 1, 0.9);
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.#{$prefix}--website-switcher--expanded {
+  width: mini-units(32);
```

Comment:
Can we change the animation to use `transform` rather than width? Transform makes the browser use the GPU, which should give a slightly smoother animation. 

```css 
.#{$prefix}--website-switcher { 
   transform: translate(16rem);
}

.#{$prefix}--website-switcher--expanded {
    transformL translate(0);
}
```